lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -349,32 +348,6 @@ def create_tar(cv, cvv, repositories, json)
           FileUtils.rm_rf(export_prefix)
         end
       end
-
-      def check_repo_type(repositories)
-        repositories.select do |repo|
-          if repo['content_type'] != 'yum'
-            raise _("The Repository '#{repo['name']}' is a non-yum repository."\
-            " Only Yum is supported at this time."\
-            " Please remove the repository from the Content View,"\
-            " republish and try the export again.")
-          end
-        end
-      end
-
-      def check_repo_download_policy(repositories)
```

Comment:
I personally like these methods staying separate, really just because it helps to document what they do. I'm perfectly fine with you moving them to the helper and creating a new method to call them. The new logic you've added could be in a third method, named by what it does.